test/passing/tag_only.ml

```diff
@@ -102,26 +103,33 @@ class b =
 
 [@@@ocamlformat "doc-comments-tag-only=fit"]
 
-open Module  (** @deprecated  *)
+open Module
+(** @deprecated  *)
```

Comment:
Why do we have a line break here ? I though `doc-comments-tag-only=fit` was for keeping theses as one-liner ?

Reply:
Must have been lost in a rebase. Fixed.